MyHashMap.java

```diff
@@ -0,0 +1,43 @@
+// Time Complexity : O(1)
+// Space Complexity : O(range of keys)
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : Tried with chaining approach, could not get it work for all edge cases
```

Comment:
Please also try implementing the way it was done in class, That solution utilizes chaining.